index.bs

```diff
@@ -2089,6 +2109,39 @@ To <dfn>await a navigation</dfn> given |context|, |request|, |wait condition|, a
 
 </div>
 
+#### The browsingContext.Locator Type #### {#type-browsingContext-Locator}
+
+[=remote end definition=] and [=local end definition=]
+
+<pre class="cddl remote-cddl local-cddl">
+browsingContext.Locator = (
+   browsingContext.CssLocator /
+   browsingContext.TextLocator /
+   browsingContext.XPathLocator
+)
+
+browsingContext.CssLocator = {
+   type: "css",
+   value: text
+}
+
+browsingContext.TextLocator {
+   type: "text",
+   value: text,
+   ? ignoreCase: bool
+   ? matchType: "full" / "partial",
+   ? maxDepth: js-uint,
```

Comment:
What is the difference between `maxDepth` here and `maxNodes` below?

Reply:
Never mind: This is explanatory from reading the algorithm below.

`maxNodes` caps the amount of returned nodes. Initially I was thinking it referred to the depth. I would argue the name could be improved (e.g. `maxNumberOfNodes` would certainly be unambiguous), feel free to bikeshed if warranted, but no strong opinions here.

Reply:
@thiagowfx I've renamed the parameter to `maxReturnedNodeCount`. Does that satisfy your concern?

Reply:
Yes, thanks! "Count" makes it unambiguous to me.